src/Test.UnitTests.Sarif/Core/StackTests.cs

```diff
@@ -38,7 +38,7 @@ public void Stack_CreateFromException()
             {
                 File.Create(Path.GetInvalidFileNameChars()[0].ToString(), 0);
             }
-            catch (ArgumentException exception)
+            catch (IOException exception)
```

Comment:
>IOException [](http://example.com/codeflow?start=19&length=11)

Change in behavior from netcoreapp3.1, File.Create now raises an IOException for illegaly chars (which makes sense).